File: vhelsing.c
#include<stdio.h>
#include<math.h>
int main()
{
	int t,i=0,r;
	long double cube;
 	long double ans;
	scanf("%d",&t);
	while(i < t)
	{
		i++;
		scanf("%d",&r);
		cube = powl(r,3);
		ans = 8*(2-sqrt(2))*cube;
		printf("%Lf\n",ans);
	}
	return 0;
}

File: vhelsing.py
from math import sqrt
def main():
	t = input()
	i = 0 
	while (i < t):
		i += 1
		r = input()
		print "%0.4f" % round(8*(2-sqrt(2))*pow(r,3),4)

if __name__ == '__main__':
	main()